ooiui/static/js/partials/StatusUIAccordion.html

```diff
@@ -0,0 +1,196 @@
+
+<% var asset = model.get('assetId'); %>
+
+<!--Remove this line when badge alerts are working-->
+<% var backgound = unknownBadge %>
+
+<!--And add this line-->
+<!--<% var backgound %>-->
+
+<% var badge %>
+
+<!-- Badge and background colors -->
+<% var blueBG = 'dogTagBtn well blue-backgroud-gradiant' %>
+<% var yellowBG = 'dogTagBtn well yellow-backgroud-gradiant ' %>
+<% var redBG = 'dogTagBtn well red-backgroud-gradiant' %>
+<% var unknownBG = 'dogTagBtn well unknown-backgroud-gradiant' %>
+
+<% var blueBadge = 'small-status-icon-ok' %>
+<% var yellowBadge = 'small-status-icon-issue' %>
+<% var redBadge =  'small-status-icon-error' %>
+<% var unknownBadge = 'small-status-icon-unknown' %>
+
+<!-- ==========================================((((((((((())))))))))))==================================================== -->
```

Comment:
This code makes me think it was "borrowed" from another project.  If so, what was the source project?